src/Chapter2.hs

```diff
@@ -740,7 +774,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains n l = filter (n `elem`) l
```

Comment:
Since you've learned about eta-reduction, you can eta-reduce this function too :slightly_smiling_face: 
```suggestion
contains n = filter (elem n)
```